scripts/e2e-test/index.js

```diff
@@ -0,0 +1,31 @@
+const { spawn } = require("child_process");
+const { readFileSync, unlinkSync } = require("fs");
+const { join } = require("path");
+
+const spawnPromise = (command, options, spawnOptions) =>
+  new Promise((resolve, reject) => {
+    const ps = spawn(command, options, {
+      stdio: "inherit",
+      ...spawnOptions,
+    });
+    ps.on("error", reject);
+    ps.on("exit", resolve);
+  });
+
+const run = async () => {
+  const envFile = "env.json";
+  await spawnPromise("node", ["get-integ-test-resources.js", envFile], {
+    cwd: __dirname,
+  });
+  const integTestResourcesEnv = JSON.parse(readFileSync(join(__dirname, envFile)).toString());
```

Comment:
Instead of writing to env.json just call `get-integ-test-resources.js` directly, and get json in return.